src/electionguard/file.c

```diff
@@ -8,47 +8,19 @@
 #include <electionguard/file.h>
 
 // Create a new file according to template
-FILE *File_new(char const *template)
+FILE *File_new(char const *path)
 {
-    bool ok = true;
-    int result_fd = -1;
-    FILE *result = NULL;
-
-    #ifdef _WIN32
-        const char *mode = "w+";
-    #else
-        const char *mode = "w+x";
-    #endif
-
-    // Duplicate the template. It needs to be mutable for mkstemp.
-    char *template_mut = strdup(template);
-    if (template_mut == NULL)
-        ok = false;
-
-    // Create and open the temporary file
-    if (ok)
-    {
-        result_fd = mkstemp(template_mut);
-        if (-1 == result_fd)
-            ok = false;
-    }
-
-    // Convert the file descriptor to a FILE*
-    if (ok)
-    {
-        result = fdopen(result_fd, mode);
-        if (result == NULL)
-            ok = false;
-    }
+    return fopen(path, "w+");

```

Comment:
should this be reverted?

Reply:
Do we even need the code in `file.h` and `file.c` anymore? I don't recall seeing them used before 🤷‍♀️

Reply:
yeah i think it can be removed